app/marsRoverPath/index.js

```diff
@@ -0,0 +1,45 @@
+var dragItem = document.querySelector('#rover-sprite');
+
+var active = false;
+var currentX;
+var currentY;
+var initialX;
+var initialY;
+var xOffset = 0;
+var yOffset = 0;
```

Comment:
When it comes to pairs of variables like this, one thing I like to do is to group them together in objects.

```
const currentMousePosition = {
    x: 0,
    y: 0
};
const initialMousePosition = {
    x: 0,
    y: 0
};
...
```

...or something. Pick names as you find appropriate

Reply:
Done